### PR TITLE
Files Added : Assignment1, assignment2, assignment3 and assignment4

### PR DESCRIPTION
**Pull request by Akash-ing**

### CODE REVIEW

**Review by vishnukopnar — APPROVED** [review]

**Review by Pranjali1999 — APPROVED** [review]

**Review by akash19972306 — APPROVED** [review]

**Review by vinod2406 — APPROVED** [review]
ok

**Review by Bhavika222 — APPROVED** [review]

**Review by Bhavika222 — APPROVED** [review]

File: Akash/html/assign 2/Assign1.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <body>
+        <h1>HTML</h1>
+        <h3>
+            Introduction
+        </h3>
+        <P>
+            HTML is standard mark up language for web pages.
+        </P>
+    </body>
+</html>
```
Comment (shivnanda6272) [inline Akash/html/assign 2/Assign1.html]: add new line

File: Akash/html/assign 2/assign2.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+     <hr>
+     <h2>Intial public offering</h2>
+     <P>Googel's intial offring (IPO)tooks place five years later,on 19 Augest 2004.<br>
+    At thats larry page,sergy engine birn,and Eric schmidt agreed to work toghster at Google for <strike>30</strike><ims>20</ims><br>
+years, unitl the year 2024.</P>
+<pre>Thank You!!</pre>
+<div>
+
+
+    <h1><center>Google</center></h1>
```
Comment (shivnanda6272) [inline Akash/html/assign 2/assign2.html]: we don't use center as a tag.older version

**Review by shivnanda6272 — CHANGES_REQUESTED** [review]

File: Akash/html/assign 4/assig2.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                border: 2px solid black;
+                border-right: 10px solid grey;
+            }
+
+            
+        </style>
+    </head>
+
+    <body>
+        <h2 style="color: rgb(155, 67, 155);"><center>Jan's Spring Term Afternoon Schedule</center></h2>
```
Comment (shivnanda6272) [inline Akash/html/assign 4/assig2.html]: don't use center as a tag.

**Review by shivnanda6272 — CHANGES_REQUESTED** [review]

**Review by shivnanda6272 — APPROVED** [review]

File: Akash/html/assign 2/Assign2.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            Assign2.html
+        </title>
+    </head>
+    <body>
+        <h1>H
+TML</h1>
```
Comment (shivling123) [inline Akash/html/assign 2/Assign2.html]: indentation not proper

File: Akash/html/assign 2/assign2.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            div{background-color: palevioletred;}
+        </style>
+        
+         <title>HTML Assign</title> -->
```
Comment (shivling123) [inline Akash/html/assign 2/assign2.html]: remove arrow infron of title

File: Akash/html/assign 2/assign4.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>assign no 4</title>
+    </head><body>
```
Comment (shivling123) [inline Akash/html/assign 2/assign4.html]: body ag should be on next line

**Review by shivling123 — CHANGES_REQUESTED** [review]
make change proper

File: Akash/html/assign 2/Assign2.html [hunk 7]
```diff
@@ -6,8 +6,8 @@
         </title>
     </head>
     <body>
-        <h1>H
-TML</h1>
+        <h1>HTML
```
Comment (SnehaMakade) [inline Akash/html/assign 2/Assign2.html]: h1 should be in a single line

File: Akash/html/assign 2/Assign2.html [hunk 8]
```diff
@@ -6,8 +6,8 @@
...
+        </h1>
         <h3>
```
Comment (SnehaMakade) [inline Akash/html/assign 2/Assign2.html]: <H3> SHOULD BE in a sigle line

File: Akash/html/assign 2/Assign2.html [hunk 9]
```diff
@@ -6,8 +6,8 @@
...
             Introduction
         </h3>
```
Comment (SnehaMakade) [inline Akash/html/assign 2/Assign2.html]: try to close the tags on the same line itself.make this change in each file

File: Akash/html/assign 2/assign2.html [hunk 10]
```diff
@@ -5,10 +5,10 @@
             div{background-color: palevioletred;}
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign2.html]: it shouldbe like div  {
                             background-color: palevioletred;
                         }
make this change in each file.

File: Akash/html/assign 2/assign2.html [hunk 11]
```diff
@@ -5,10 +5,10 @@
...
         
-         <title>HTML Assign</title> -->
+         <title>HTML Assign</title> 
     </head>
     <body>
-        <h1><center>Google</center></h1>
+        <h1 style="text-align: center;">Google</h1>
       
       <p><i>This articel is about the company.</i></p>
       <h2>From the grage to the Googleplex</h2>
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign2.html]: always add aspace after colon and comma so make this change everywhere

File: Akash/html/assign 2/assign2.html [hunk 12]
```diff
@@ -30,7 +30,7 @@ <h2>Intial public offering</h2>
 <div>
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign2.html]: remove extra lines

File: Akash/html/assign 2/assign2.html [hunk 13]
```diff
@@ -45,7 +45,7 @@ <h2>Intial public offering</h2>
     <P>Googel's intial offring (IPO)tooks place five years later,on 19 Augest 2004.<br>
         At thats larry page,sergy engine birn,and Eric schmidt agreed to work toghster at Google for <strike>30</strike><ims>20</ims>
     years, unitl the year 2024.</P>  
-      <h3><i><center>Thank You !!</center></i></h3>
+      <h3 style="text-align: center;"><i>Thank You !!</i></h3>
 </div>
     </body>
 </html>
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign2.html]: add a new end line in each file

File: Akash/html/assign 2/assign4.html [hunk 14]
```diff
@@ -2,12 +2,14 @@
 <html>
     <head>
         <title>assign no 4</title>
-    </head><body>
+    </head>
+    <body>
         <h1>My Self</h1>
         <h3>Akki Rajput</h3>
         <hr>
         <p>I am akash ingle.i am from chikhali,Dist buldana.<br>
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign4.html]: add a space after comma

File: Akash/html/assign 2/assign4.html [hunk 15]
```diff
@@ -2,12 +2,14 @@
...
         My education is BCA(Bachelor of computer application)in last samester <br>
-        I am studing in Aurangabad (sambhaji nager)</p>
+        I am studing in Aurangabad (sambhaji nager)
+        </p>
     </body>
 </html>
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign4.html]: add a new end line

File: Akash/html/assign 4/assig2.html [hunk 16]
```diff
@@ -24,7 +24,7 @@
     </head>
 
     <body>
-        <h2 style="color: rgb(155, 67, 155);"><center>Jan's Spring Term Afternoon Schedule</center></h2>
+        <h2 style="color: rgb(155, 67, 155);text-align: center;">Jan's Spring Term Afternoon Schedule</h2>
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assig2.html]: remove extra lines in this file and a space after colon and before opening bracket there should be a space so add tht and add a space after semicolon

File: Akash/html/assign 4/assig2.html [hunk 17]
```diff
@@ -24,7 +24,7 @@
...
         
         <table style="width:100%;">
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assig2.html]: add aspace after colon

File: Akash/html/assign 4/assig2.html [hunk 18]
```diff
@@ -24,7 +24,7 @@
...
             <tr class="a">
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assig2.html]: there is no class inside style tag of name a so if this class is not in use then remove it and remove all extra lines

File: Akash/html/assign 5/assign1.html [hunk 19]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            body{
```
Comment (SnehaMakade) [inline Akash/html/assign 5/assign1.html]: add a space before opening bracket

File: Akash/html/assign 5/assign1.html [hunk 20]
```diff
@@ -0,0 +1,54 @@
...
+                background-image: url("index.jpeg");
+            }
+            
+            p{
+               background-color: orange;
+            }
+            ul{
+                background-color: orange;
+            }
+
```
Comment (SnehaMakade) [inline Akash/html/assign 5/assign1.html]: remove all extra lines

File: Akash/html/assign 5/assign1.html [hunk 21]
```diff
@@ -0,0 +1,54 @@
...
+            }
+            
+               
+            
+
+        </style>
+
+    </head>
+    <body>
+        <h1>
```
Comment (SnehaMakade) [inline Akash/html/assign 5/assign1.html]: make  a h1 in a single line

File: Akash/html/assign 5/assign1.html [hunk 22]
```diff
@@ -0,0 +1,54 @@
...
+            Akki
+        </h1>
+        <hr>
+        <h3 style="text-align: right;"><b><i>About Me</i></b></h3>
+
```
Comment (SnehaMakade) [inline Akash/html/assign 5/assign1.html]: remove all extra lines

File: Akash/html/assign 5/assign1.html [hunk 23]
```diff
@@ -0,0 +1,54 @@
...
+                </li>
+                <li>
+                    Photo Gallery
+                </li>
+            </ul>
+            
+            
+        </p>
+    </body>
+</html>
```
Comment (SnehaMakade) [inline Akash/html/assign 5/assign1.html]: add a new end line in all files and remove extra lines in all files

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do changes as per comments given and change the pr name, give name according to assignments given

File: Akash/html/assign 2/Assign2.html [hunk 24]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
```
Comment (SnehaMakade) [inline Akash/html/assign 2/Assign2.html]: write title tag in a single line

File: Akash/html/assign 2/Assign2.html [hunk 25]
```diff
@@ -0,0 +1,19 @@
...
+            Assign2.html
+        </title>
+    </head>
+    <body>
+        <h1>HTML</h1>
+        <h3>Introduction</h3>
+        <p>
```
Comment (SnehaMakade) [inline Akash/html/assign 2/Assign2.html]: include p tag in a same line.

File: Akash/html/assign 2/Assign2.html [hunk 26]
```diff
@@ -0,0 +1,19 @@
...
+            HTML is a standard markup language for web pages.
+        </p>
+        <h5>Notepad or textedit</h5>
+        <p>Web paged can be created and modified by using professional HTML editors.<br>
+           However,for learning HTML we recommend a simple text editor like Notepad(pc)or textedit(Mac).</p>
+    </body>
+</html> 
+
```
Comment (SnehaMakade) [inline Akash/html/assign 2/Assign2.html:17]: check is there any trailing spaces present in line no. 19 as its showing the error of no new line at end of file

File: Akash/html/assign 2/assign2.html [hunk 27]
```diff
@@ -0,0 +1,72 @@
...
+        </style>
+        <title>HTML Assign</title>
+    </head>
+    <body>
+        <h1 style="text-align: center">Google</h1>
+        <p><i>This articel is about the company.</i></p>
+        <h2>From the grage to the Googleplex</h2>
+        <h3>
+            The Google story begins in 1995 at<b>Stanford university.</b><br />
+            <i><b>Larry page</b></i
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign2.html]: check indentation </i> should be in a single line

File: Akash/html/assign 2/assign2.html [hunk 28]
```diff
@@ -0,0 +1,72 @@
...
+            >was considearing stanford for the gard school and<i><b>Sargey brain,</b></i
+            >a student<br />
+            there,was assigned to show him around.
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign2.html]: always add a space after comma

File: Akash/html/assign 2/assign2.html [hunk 29]
```diff
@@ -0,0 +1,72 @@
...
+        </p>
+        <pre>Thank You!!</pre>
+        <div>
+            <h1 style="text-align: center">Google</h1>
+            <br />
+            <p><i>This articel is about the company.</i></p>
+            <h2>From the grage to the Googleplex</h2>
+            <h3>
+                The Google story begins in 1995 at<b>Stanford university.</b><br />
+                <i><b>Larry page</b></i
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign2.html]: do proper closing of i tag

File: Akash/html/assign 2/assign2.html [hunk 30]
```diff
@@ -0,0 +1,72 @@
...
+                >was considearing stanford for the gard school and<i><b>Sargey brain,</b></i
+                >a student there, was assigned to show him around.
+            </h3>
+            <p>
+                By some accounts,they dissagred about nearly everything during that frist meeting,but by the following year thry struck a
+                <mark>patnership</mark>.<br />
+                Working about their drom rooms, they built a search engine thats used links to determine the importance of individual pages on the work.<a
+                    href="https://www.World Wide web/"
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign2.html]: check indentation

File: Akash/html/assign 2/assign2.html [hunk 31]
```diff
@@ -0,0 +1,72 @@
...
+                    >World Wide Web</a
+                >
+                <br />
+            </p>
+            <p>
+                They called this search engine mark <i><marki>Backrub</marki></i>
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign2.html]: check for the tag u have used it should be < mark> only

File: Akash/html/assign 3/Assign9.html [hunk 32]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (SnehaMakade) [inline Akash/html/assign 3/Assign9.html]: where is doctype ? add that

File: Akash/html/assign 3/Assign9.html [hunk 33]
```diff
@@ -0,0 +1,36 @@
...
+<html>
+    <head>
+        <style>
+            h1{text-align: center;}
```
Comment (SnehaMakade) [inline Akash/html/assign 3/Assign9.html]: there should be space before opening bracket like h1 {} not like h1{}

File: Akash/html/assign 3/Assign9.html [hunk 34]
```diff
@@ -0,0 +1,36 @@
...
+            p{text-align: center;}
+            .p1 {
+                font-family: "Times New Roman", Times, serif;
+            }
+        </style>
+    </head>
+    <body style="background-color: pink;">
+        <h1 class="p1">Ambersoft</h1><br>
+
```
Comment (SnehaMakade) [inline Akash/html/assign 3/Assign9.html]: remove extra lines

File: Akash/html/assign 3/Assign9.html [hunk 35]
```diff
@@ -0,0 +1,36 @@
...
+        <p>
+            
+            <a href="assign8.html">Home | </a>
+            <a href="Assign9.html"> MI-MEAN | </a>
+            <a href="assign10.html"> Contact | </a>
+         </p><br>
+
```
Comment (SnehaMakade) [inline Akash/html/assign 3/Assign9.html]: remove all extra lines

File: Akash/html/assign 3/assign10.html [hunk 36]
```diff
@@ -0,0 +1,21 @@
+<html>
```
Comment (SnehaMakade) [inline Akash/html/assign 3/assign10.html:2]: add a doctype.

File: Akash/html/assign 3/assign10.html [hunk 37]
```diff
@@ -0,0 +1,21 @@
...
+<head>
+	<title></title>
```
Comment (SnehaMakade) [inline Akash/html/assign 3/assign10.html]: if there is no title then remove this tag or else add a title .

File: Akash/html/assign 3/assign10.html [hunk 38]
```diff
@@ -0,0 +1,21 @@
...
+    <link rel="stylesheet" href="style.css"></link>
+</head>
+
+<body style="background-color: pink;">
```
Comment (SnehaMakade) [inline Akash/html/assign 3/assign10.html]: check indentation

File: Akash/html/assign 3/assign8.html [hunk 39]
```diff
@@ -0,0 +1,35 @@
+<html>
+    <head>
+        <title>assignment3</title>
+        <link rel="stylesheet" href="style.css"></link>
+    </head>
+
```
Comment (SnehaMakade) [inline Akash/html/assign 3/assign8.html]: remove extra line

File: Akash/html/assign 3/assign8.html [hunk 40]
```diff
@@ -0,0 +1,35 @@
...
+		    This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.
+		    This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.
+		    This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.
+		    This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the Home Page.This is the 
+        </div>
+        <br>
+        <h4>
```
Comment (SnehaMakade) [inline Akash/html/assign 3/assign8.html]: include h4in a single line

File: Akash/html/assign 3/assign8.html [hunk 41]
```diff
@@ -0,0 +1,35 @@
...
+            Home page subheading
+        </h4>
+        <p>
+            This is Home Page Subheading.This is Home Page Subheading.This is Home Page Subheading.This is Home Page Subheading.This is Home Page Subheading.This is Home Page Subheading.This is Home Page Subheading.This is Home Page Subheading.This is Home Page Subheading.This is Home Page Subheading.This is Home Page Subheading.This is Home Page Subheading.
+        </p>
+
```
Comment (SnehaMakade) [inline Akash/html/assign 3/assign8.html]: remove extra line

File: Akash/html/assign 3/style.css [hunk 42]
```diff
@@ -0,0 +1,15 @@
+#left{
```
Comment (SnehaMakade) [inline Akash/html/assign 3/style.css]: always add a space before opening bracket and a line between each block

File: Akash/html/assign 4/assig2.html [hunk 43]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            table,
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assig2.html]: include table and th in a single line

File: Akash/html/assign 4/assig2.html [hunk 44]
```diff
@@ -0,0 +1,82 @@
...
+            td {
+                border: 10px solid grey;
+                border-collapse: collapse;
+            }
+
+            th,
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assig2.html]: include th td in a single line

File: Akash/html/assign 4/assig2.html [hunk 45]
```diff
@@ -0,0 +1,82 @@
...
+
+            td {
+                border: 2px solid black;
+                border-right: 10px solid grey;
+            }
+        </style>
+    </head>
+    <body>
+        <h2 style="color: rgb(155, 67, 155); text-align: center">Jan's Spring Term Afternoon Schedule</h2>
+
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assig2.html]: remove all extra lines

File: Akash/html/assign 4/assign1.html [hunk 46]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            .p1 {
+                font-family: "cursive";
+            }
+            table,
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assign1.html]: include table th and td in a single line

File: Akash/html/assign 4/assign3.html [hunk 47]
```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            table,
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assign3.html]: include all tags in a single line

File: Akash/html/assign 4/assign3.html [hunk 48]
```diff
@@ -0,0 +1,74 @@
...
+            <tr>
+                <th rowspan="3">Day</th>
+                <th colspan="3">Seminar</th>
+            </tr>
+
+            <tr>
+                <th colspan="2">Schedule</th>
+                <th rowspan="2">Topic</th>
+            </tr>
+
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assign3.html]: remove all extra lines

File: Akash/html/assign 5/assign1.html [hunk 49]
```diff
@@ -0,0 +1,43 @@
...
+        <style>
+            body {
+                background-color: coral;
+                width: 1000px;
+                margin: 300px auto;
+                padding: 20px;
+                border-style: solid;
+                background-image: url("index.jpeg");
+            }
+
```
Comment (SnehaMakade) [inline Akash/html/assign 5/assign1.html]: remove extra line

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do the changes as per comments given and change pr name also

**Review by vishnukopnar — APPROVED** [review]

File: Akash/html/assign 2/assign2.html [hunk 50]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign2.html]: why not removed this extra line?

File: Akash/html/assign 2/assign2.html [hunk 51]
```diff
@@ -0,0 +1,76 @@
...
+<head>
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign2.html]: indentation is not proper

File: Akash/html/assign 2/assign2.html [hunk 52]
```diff
@@ -0,0 +1,76 @@
...
+    <style>
+        div {
+            background-color: palevioletred;
+        }
+    </style>
+    <title>HTML Assign</title>
+</head>
+
+<body>
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign2.html]: do proper indentation

File: Akash/html/assign 2/assign2.html [hunk 53]
```diff
@@ -0,0 +1,76 @@
...
+            At thats larry page, sergy engine birn, and Eric schmidt agreed to work toghster at Google for
+            <strike>30</strike>
+            <ims>20</ims> years, unitl the
+            year 2024.
+        </p>
+        <h3 style="text-align: center"><i>Thank You !!</i></h3>
+    </div>
+</body>
+
+</html>
```
Comment (SnehaMakade) [inline Akash/html/assign 2/assign2.html]: add a new end line

File: Akash/html/assign 3/assign10.html [hunk 54]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (SnehaMakade) [inline Akash/html/assign 3/assign10.html]: indentation is not proper

File: Akash/html/assign 3/assign7.html [hunk 55]
```diff
@@ -0,0 +1,81 @@
...
+            }
+            div {
+                color: red(240, 6, 6, 0.192);
+            }
+            div1 {
+                color: tomato;
+            }
+        </style>
+    </head><body>
+        <h1><p class="p1">Ambersoft</p></h1>
```
Comment (SnehaMakade) [inline Akash/html/assign 3/assign7.html]: why to use p tag inside h1 tag instead use h1 tag only

File: Akash/html/assign 3/assign7.html [hunk 56]
```diff
@@ -0,0 +1,81 @@
...
+                list-style-position: inside;
+            }
+            div {
+                color: red(240, 6, 6, 0.192);
+            }
+            div1 {
+                color: tomato;
+            }
+        </style>
+    </head><body>
```
Comment (SnehaMakade) [inline Akash/html/assign 3/assign7.html]: what kind of indentation is this? do proper indentation

File: Akash/html/assign 3/assign7.html [hunk 57]
```diff
@@ -0,0 +1,81 @@
...
+        <h1><p class="p1">Ambersoft</p></h1>
+        <div>
+            <ol class="d"><num>1.</num>MI-MEAN
+            <ol class="c">
+                <li>MI-MEAN-6</li>
+            <ul>​
```
Comment (SnehaMakade) [inline Akash/html/assign 3/assign7.html]: indentation?

File: Akash/html/assign 3/assign7.html [hunk 58]
```diff
@@ -0,0 +1,81 @@
...
+                <li>pallavi</li>
+                <li>reva</li>   
+            </ul>
+            
+            <li class="b">MI-MEAN-9</li>
+
+              <ul class="a">
+                <li>Anil</li>
+                <li>Monaj</li>
+
```
Comment (SnehaMakade) [inline Akash/html/assign 3/assign7.html]: remove all extra lines

File: Akash/html/assign 4/assign1.html [hunk 59]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            .p1 {
+                font-family: "cursive";
+            }
+            table,th, td {
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assign1.html]: add space after comma

File: Akash/html/assign 4/assign1.html [hunk 60]
```diff
@@ -0,0 +1,43 @@
...
+                border: 2px solid black;
+                border-collapse: collapse;
+            }
+        </style>
+    </head>
+    <body>
+        <margin class="p1"
+            ><center><b>A test table with merged cells</b></center></margin
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assign1.html]: dont uise center tag instead use text-align: center property

File: Akash/html/assign 4/assign3.html [hunk 61]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            table,th,td {
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assign3.html]: add space after comma

File: Akash/html/assign 4/assign3.html [hunk 62]
```diff
@@ -0,0 +1,65 @@
...
+                border: 2px solid grey;
+            }
+
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assign3.html]: remove extra line and add space after comma

File: Akash/html/assign 5/assign1.html [hunk 63]
```diff
@@ -0,0 +1,42 @@
...
+    <body>
+        <h1>Akki</h1>
+        <hr>
+        <h3 style="text-align: right;"><b><i>About Me</i></b></h3>
+         <p>
+            i am akash ingle,<br>
+            i am from chikhali,Maharastra,<br>
+         <h3 style="text-align: right;"><b><i>More about me</i></b></h3>
+            <ul style="color: blue;">
+                <li>
```
Comment (SnehaMakade) [inline Akash/html/assign 5/assign1.html]: include <li> tag in a single line only

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do changes as per comments given

File: Akash/html/assign 3/assign10.html [hunk 64]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+    <head>
+	   <link rel="stylesheet" href="style.css"></link>
+    </head>
+    <body style="background-color: pink;">
+	    <h2>Ambersoft</h2>
```
Comment (SnehaMakade) [inline Akash/html/assign 3/assign10.html:7]: indentation is not proper

File: Akash/html/assign 4/assig2.html (hunk 44 above)
Comment (SnehaMakade) [inline Akash/html/assign 4/assig2.html]: havent done this change why? write it as table, td {}

File: Akash/html/assign 4/assign1.html [hunk 65]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            .p1 {
+                font-family: "cursive";
+                text-align: center;
+            }
+
+            table,
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assign1.html]: do this change

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do the changes as per comments given

File: Akash/html/assign 3/assign8.html [hunk 66]
```diff
@@ -0,0 +1,32 @@
+<html>
```
Comment (SnehaMakade) [inline Akash/html/assign 3/assign8.html:2]: no doctype is present add that

File: Akash/html/assign 4/assig2.html [hunk 67]
```diff
@@ -0,0 +1,73 @@
...
+    <head>
+        <style>
+              table, th, td { border: 10px solid grey;
+                              border-collapse: collapse;
+                              text-align: center;
+                              border-bottom: 2px solid black; 
+                              border-right: 10px solid grey; 
+                              border: 2px solid black;
+                              border-right: 10px solid grey;
+                            }
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assig2.html]: this bracket should be below tables tmeansclosing bracket should be on the same line

File: Akash/html/assign 4/assign3.html [hunk 68]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            table,
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assign3.html]: why changes not done in this file

File: Akash/html/assign 4/assign3.html [hunk 69]
```diff
@@ -0,0 +1,66 @@
...
+            th,
+            td {
+                border: 2px solid grey;
+            }
+
+            th,
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assign3.html]: do proper changes

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do proper changes

File: Akash/html/assign 4/assign3.html [hunk 70]
```diff
@@ -2,17 +2,11 @@
...
-            td {
-                border: 2px solid grey;
-            }
-
-            th,
-            td {
-                text-align: center;
-                border: 2px solid black;
-            }
+            table,
```
Comment (SnehaMakade) [inline Akash/html/assign 4/assign3.html]: this too also should be in a single line like table, th, td { watever style u want to write} in this way

**Review by SnehaMakade — APPROVED** [review]

**Pull request merged**